src/dsp_tools/cli/call_action.py

```diff
@@ -182,11 +182,12 @@ def _call_xmlupload(args: argparse.Namespace) -> bool:
         return validate_xml(args.xmlfile)
     else:
         interrupt_after = args.interrupt_after if args.interrupt_after > 0 else None
+        iiif_validation = args.no_iiif_uri_validation is False
```

Comment:
I think it would be good to add a test for this in `test/unittests/cli/test_cli_with_mock.py`, because I'm not 100% sure if this works as intended

Reply:
I decided to make the validation an opt-out with a flag, I am happy to rename it, or change it into an opt-in.

Reply:
I'm not against the opt-out, I'm struggling to understand this expression. Is `args.no_iiif_uri_validation` a boolean? Is it False or None if the user doesn't provide anything? Why `x is False` not `x == False` or `!x`?

Reply:
Ah now I understand, this turns the presence or absence of the arg into a boolean. It is a copilot suggestion, but I can change it.